pkg/reconciliation/reconcile_datacenter.go

```diff
@@ -125,6 +150,14 @@ func (rc *ReconciliationContext) deletePVCs() error {
 		"numPVCs", len(persistentVolumeClaimList))
 
 	for _, pvc := range persistentVolumeClaimList {
+
+		if isBeingUsed, err := rc.isBeingUsed(pvc); err != nil {
+			logger.Error(err, "Failed to check if PVC is being used")
+			return err
+		} else if isBeingUsed {
+			return fmt.Errorf("PersistentVolumeClaim %s is still being used by a pod", pvc.Name)
```

Comment:
Question: Upon ordinarily deleting a k8ssandra cluster, I've seen a bunch of ERROR lines caused by this in the operator logs. I understand it's a normal pattern to error out until a condition (in this case pod existence) changes. But the "ERROR	Reconciler error" seems a bit harsh. Perhaps we can do something to soften this?

Reply:
That logging comes from the controller-runtime, so there's not much we can do about it. A warn in the case of reconciler error which retry might fix makes sense, but I'm not aware of having that ability. 

In the end, it's still an error. It's not like we would log this at "FATAL".